### ISSUE TITLE
subplot segmentation fault

### ISSUE THREAD

**Issue by seisman** [GenericMappingTools/gmt#1332]
The script now fails with a segmentation fault.
```
gmt begin panels pdf
  gmt subplot begin 2x2 -Fs8c -M5p -A -SCb -SRl -Bwstr
    gmt subplot set
    gmt basemap -R0/80/0/10
    gmt subplot set
    gmt basemap
    gmt subplot set
    gmt basemap
    gmt subplot set
    gmt basemap
  gmt subplot end
gmt end
```

**Comment by PaulWessel** [comment]
Wast that ever a valid syntax, i.e., using subplot set with no arguments? I guess you mean this to be the equivalent to -c on the next command instead?  I will see what I can do.  I can certaintly at the very least give an error.

**Comment by seisman** [comment]
It was valid in 6.0.0rc3.

**Comment by PaulWessel** [comment]
Sorry, fixed it and then committed straight to 6.0 by mistake.  But show work fine.  Let me know if anything else needs to be done.

**Comment by seisman** [comment]
It works.
